silta/silta.yml

```diff
@@ -30,6 +30,7 @@ php:
       drush eshd -y
       drush eshs
       drush mim --group=demo_content --execute-dependencies --skip-progress-bar
+      drush cron
```

Comment:
This should make sure that the file exists when the starterkit is deployed in silta